### DISCUSSION TITLE
Computing variable importance (vimp)

### DISCUSSION THREAD

**Discussion by ykim1296** [kogalur/randomForestSRC#110] [Q&A] [answered]
Thank you @kogalur and @ishwaran for the amazing package. I have a question regarding variable importance. I compared variable importance result from randomForest package (univariate) and from randomForestSRC package (multivariate). I used permutation method for both of them, but the numbers are much different. For randomForest package, I got numbers ranging from 50-300 but with randomForestSRC, ranges were 0-0.03. Could you give me more information about how vimp computes variable importance (especially with the 'permutation' option)?

**Comment by ishwaran** [comment, marked as answer]
See this vignette (https://luminwin.github.io/randomForestSRC/articles/rfsrc-subsample.html) for a description of how variable importance (VIMP) is calculated.
